e2e/multi-env-app/multi_env_suite_test.go

```diff
@@ -15,17 +15,13 @@ import (
 
 var cli *client.CLI
 var appName string
-var testEnvironmentProfile string
-var prodEnvironmentProfile string
```

Comment:
were these unused?

Reply:
they were used as in
```go
// In multi_env_suite_test.go
var testEnvironmentProfile string
var _ = BeforeSuite(func() {
	testEnvironmentProfile = "e2etestenv"
}

// In multi_env_test.go
_, err := copilot.EnvInit(&client.EnvInitRequest{
    AppName: appName,
    EnvName: "test",
    Profile: testEnvironmentProfile,
})
```

But since now the profile's name is always the same as the env name, we can just write 
```go
// In multi_env_test.go
_, err := copilot.EnvInit(&client.EnvInitRequest{
    AppName: appName,
    EnvName: "test",
    Profile:"test",
})
```
replacing `testEnvironmentProfile` with a plain sting `"test"`! So the var is no longer needed.